libraries/core_libs/network/src/tarcap/stats/packets_stats.cpp

```diff
@@ -1,35 +1,56 @@
 #include "network/tarcap/stats/packets_stats.hpp"
 
+#include "json/writer.h"
+
 namespace taraxa::network::tarcap {
 
+/**
+ * @param value
+ * @return unstyled json string (without new lines and whitespaces).
+ */
+std::string jsonToUnstyledString(const Json::Value& value) {
+  Json::StreamWriterBuilder builder;
+  builder["indentation"] = "";  // assume default for comments is None
+  return Json::writeString(builder, value);
+}
+
 PacketsStats::PacketsStats(const addr_t& node_addr) { LOG_OBJECTS_CREATE("NETPER"); }
 
-void PacketsStats::addReceivedPacket(const dev::p2p::NodeID& node, const std::string& packet_type,
-                                     const SinglePacketStats& packet) {
+void PacketsStats::addReceivedPacket(const std::string& packet_type, const SinglePacketStats& packet) {
   received_packets_stats_.addPacket(packet_type, packet);
 
-  LOG(log_dg_) << "(\"" << node << "\") received " << packet_type << " packet from (\"" << packet.node_
-               << "\"). Stats: " << packet;
+  auto packet_json = packet.getStatsJson();
+  packet_json["type"] = packet_type;
+  LOG(log_tr_) << "Received packet: " << jsonToUnstyledString(packet_json);
```

Comment:
If I understand correctly addReceivedPacket and addSentPacket are called for every received/sent packet, maybe we should check if this change hurts performance when receiving/sending large number of packets. Even if this log channel is off these json methods are called which might affect performance

Reply:
this is called inside packet handlers, which are running & processing packets on separate threads, which means that printing out json representation of packet is basically negligible when compared to all the code that is processed inside specific packet handlers....

Reply:
I still think that there is a possibility that some minor impact on performance is possible. Considering the number of packets we might be receiving I would go with some binary struct and not using json internally to log this. There is json processing inside of packet.getStatsJson and then call to jsonToUnstyledString.

Reply:
Ok I did spend some time on it and no you are wrong, json creation has negligible impact on performance. Simple string creation takes about 5 us, json string creation takes around 25 us. Then  just logging any string takes 50 us, so it is 75 us for logging json string vs 55 us for logging simple packet string.

Reply:
I wouldn't count in the 50us for logging. I assume we would not log this always. If we wanted a node with max performance this log would not be on but still this code would run for every packet even with the log channel off and then 5us vs 25us if we might have thousands of packets per second in my opinion is not that negligible. I don't think this will cause some serious performance issues but I think it would be more optimal to go with some native struct and not json for these logs.

Reply:
json is removed